### ISSUE TITLE
XCode 12 UIApplicationDelegate docs are outdated

### ISSUE THREAD

**Issue by StanfordLin** [firebase/firebase-ios-sdk#6575]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12.0.1
  * Firebase SDK version: latest based off pod init
  * Firebase Component: Core
  * Component version: n/a
  * Installation method: CocoaPods 

#### Steps to reproduce:

What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

If you have a downloadable sample project that reproduces the bug you're reporting, you will
likely receive a faster response on your issue.

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by StanfordLin** [comment]
I've also tried using

**Comment by ryanwilson** [comment]
Thanks for the report - we definitely need to update those docs.

The way you had mentioned should definitely work, and the AppDelegateAdapter is necessary for features like Messaging. If you don't need any features that require the adapter, Peter Friese's recommendation works as well. I'll chat with the docs team to ensure we get this updated, thanks!

**Comment by ryanwilson** [comment]
cc @peterfriese

**Comment by morganchen12** [comment]
Yep, this one seems fixed to me. Googlers see cl/449035125 and cl/449990337.

If there's anything missing @StanfordLin please re-open or file a new issue.
